views/account/settings.hbs

```diff
@@ -243,7 +241,7 @@
 							</table>
 						{{else}}
 							<h3 class=" h5 text-muted text-center">
-								<span>Keine Drittanbieter genutzt</span>
+								<span>{{$t "account.text.nothirdPartyProvidersUsed" }}</span>
```

Comment:
account.text.nothirdPartyProvidersUsed -> noThirdPartyProvidersUsed

Reply:
Thanks for the observation. it's being changed.